.github/container/Dockerfile

```diff
@@ -1,9 +1,10 @@
-FROM debian:bullseye
+FROM debian:bookworm
 
 RUN dpkg --add-architecture arm64 && \
     apt-get update && \
     apt-get full-upgrade --no-install-recommends -y \
         build-essential crossbuild-essential-arm64 default-jdk-headless git \
-        cmake debhelper devscripts lintian swig pkgconf:arm64 python3-dev:arm64 libjson-c-dev:arm64 && \
+        cmake debhelper devscripts lintian swig pkgconf:arm64 python3-dev:arm64 libjson-c-dev:arm64 libstdc++6:arm64 && \
+    ln -s /usr/bin/python3 /usr/bin/python && \
```

Comment:
python-is-python3依赖冲突的替代方案

Reply:
为什么不直接用python-is-python3

Reply:
> 为什么不直接用python-is-python3

https://github.com/radxa-pkg/mraa/pull/2/files#r1746676203

Reply:
所以为什么不直接用python-is-python3？依赖冲突是怎么回事

Reply:
```
[0 ~/Documents/GitHub/mraa]$ .github/container/build-deb 
STEP 1/2: FROM debian:bookworm
STEP 2/2: RUN dpkg --add-architecture arm64 &&     apt-get update &&     apt-get full-upgrade --no-install-recommends -y         build-essential crossbuild-essential-arm64 default-jdk-headless git         cmake debhelper devscripts lintian swig pkgconf:arm64 python-is-python3 python3-dev:arm64 libjson-c-dev:arm64 libstdc++6:arm64 &&     ln -s /usr/bin/python3 /usr/bin/python &&     adduser --gecos runner --disabled-password runner &&     rm -rf /var/lib/apt/lists/*
Get:1 http://deb.debian.org/debian bookworm InRelease [151 kB]
Get:2 http://deb.debian.org/debian bookworm-updates InRelease [55.4 kB]
Get:3 http://deb.debian.org/debian-security bookworm-security InRelease [48.0 kB]
Get:4 http://deb.debian.org/debian bookworm/main amd64 Packages [8787 kB]
Get:5 http://deb.debian.org/debian bookworm/main arm64 Packages [8689 kB]
Get:6 http://deb.debian.org/debian bookworm-updates/main amd64 Packages [13.8 kB]
Get:7 http://deb.debian.org/debian bookworm-updates/main arm64 Packages [13.7 kB]
Get:8 http://deb.debian.org/debian-security bookworm-security/main arm64 Packages [176 kB]
Get:9 http://deb.debian.org/debian-security bookworm-security/main amd64 Packages [179 kB]
Fetched 18.1 MB in 3s (6206 kB/s)
Reading package lists...
Reading package lists...
Building dependency tree...
Reading state information...
Calculating upgrade...
Some packages could not be installed. This may mean that you have
requested an impossible situation or if you are using the unstable
distribution that some required packages have not yet been created
or been moved out of Incoming.
The following information may help to resolve the situation:

The following packages have unmet dependencies:
 python3-dev:arm64 : Depends: python3:arm64 (= 3.11.2-1+b1) but it is not installable
 python3.11-dev:arm64 : Depends: python3.11:arm64 (= 3.11.2-6+deb12u3) but it is not installable
E: Unable to correct problems, you have held broken packages.
Error: building at STEP "RUN dpkg --add-architecture arm64 &&     apt-get update &&     apt-get full-upgrade --no-install-recommends -y         build-essential crossbuild-essential-arm64 default-jdk-headless git         cmake debhelper devscripts lintian swig pkgconf:arm64 python-is-python3 python3-dev:arm64 libjson-c-dev:arm64 libstdc++6:arm64 &&     ln -s /usr/bin/python3 /usr/bin/python &&     adduser --gecos runner --disabled-password runner &&     rm -rf /var/lib/apt/lists/*": while running runtime: exit status 100
[chen@ChenArchLinux 18:39]
[100 ~/Documents/GitHub/mraa]$ 
```

Reply:
参照[zfs](https://github.com/radxa-pkg/zfs/blob/master/.github/container/equivs-dummy.equivs#L3)提供fake dependency试试？